### PR TITLE
Add SNS options for update subcommand

### PR DESCRIPTION
**Pull request by a1exsh**
It works the same way as in create subcommand.  If you don't specify an SNS
topic, the new alarm created for autorecovery will not trigger notification.

### CODE REVIEW

**Comment by a1exsh** [comment]
:+1:

File: planb/update_cluster.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
 from .common import ec2_client, \
     dump_dict_as_file, load_dict_from_file, \
     dump_user_data_for_taupage, list_instances, \
+    override_ephemeral_block_devices, \
+    setup_sns_topics_for_alarm, create_auto_recovery_alarm, \
     override_ephemeral_block_devices, create_auto_recovery_alarm, \
     create_instance_profile, get_instance_profile
```
Comment (slitsche) [inline planb/update_cluster.py]: It looks like 2 functions are imported twice.

Comment (a1exsh) [inline planb/update_cluster.py]: Huh...  and that looked like a sooo simple merge conflict. ;)

File: planb/update_cluster.py [hunk 2]
```diff
@@ -292,10 +294,15 @@ def configure_instance(ec2: object, volume: dict, options: dict):
...
+    if options['alarm_topics']:
+        alarm_sns_topic_arn = options['alarm_topics'][region]
+
+    create_auto_recovery_alarm(region,
                                options['cluster_name'],
                                instance_id,
-                               None) # TODO: get SNS topic somewhere
+                               alarm_sns_topic_arn)
 
     # TODO: we should have another transition to wait for Cassandra to
```
Comment (slitsche) [inline planb/update_cluster.py]: This comment could be removed.

Comment (a1exsh) [inline planb/update_cluster.py]: Good point, but that PR is merged already.

**Comment by slitsche** [comment]
👍

File: planb/update_cluster.py (hunk 2 above)
Comment (slitsche) [inline planb/update_cluster.py]: That's why we could remove it here.
Still open

**Comment by a1exsh** [comment]
:+1:

**Pull request merged**